docs/testnet-guardian-runbook.md

```diff
@@ -0,0 +1,152 @@
+
+##### Check docs
+
+Verify that online docs https://codaprotocol.com/docs/getting-started are updated and user journeys are consistent with the release.
+
+##### Monitor the QA-net and report status
+
+Monitor the health of the QA-net by checking the logs (https://console.cloud.google.com/kubernetes), grafana dashboard (https://o1testnet.grafana.net), telemetry and other tools. Check for
+1. Crashes
+2. Forks
+3. Expected TPS
+4. Blocks being produced consistently
+
+Report the status of the network and testing to the rest of the team on `#qa-net-discussion` channel.
+
+On completing the QA-net phase successfully, let the infrastructure team know the same and tear down the QA-net before the release.
+
+Easy peasy?
+
+Maybe not, sometimes there could be unintended twists in the story and the testnet guardian might encounter issues along the way. Following is the list of possible situations and actions that the testnet guardian may take:
+
+| | Event | Action | Criticality | Owner(s)|
+|--|------|--------|-------------|--|
+|1| Unsuccessful build jobs. (We shouldn't be merging hotfixes before all the CI jobs are successful).| Sometimes rerunning the build job fixes it. For example, at times docker upload fails and passes on rerun. But if there are other code/ci config related issues then the QA-net cannot continue until the issue is either reverted or fixed. In the case of a hotfix for issues encountered during QA-net, the fix should be reverted| Blocker| |
+|2| Configuration/infra issues when deploying a QA-net | There are some troubleshooting notes in the notion document for the instructions to deploy a network. If that doesn't help then contact the any of the members listed in owners column for help | Blocker | Conner, Ahmad, Nathan|
+| 3| Unstable Testnet* (criteria for an unstable testnet is listed below)| Create an issue describing the observed behavior. The protocol team would need to investigate the problem and provide a hotfix before proceeding further with the release | Blocker | Aneesha |
+| 4| Errors when completing the challenges| Create an issue describing the observed behavior. Might have to exclude the challenge that causes the error if it cannot be patched and tested within the QA-net phase. Let the marketing and communications team know? | Major | Aneesha, Brandon, Christine|
+| 5 | Other errors/suspicious behavior observed when monitoring. These don't destabilize the testnet or affect the challenges | Create an issue describing the behavior | Minor | |
+| 6 | Patch/hotfix | On receiving hotfix(es) on release and master(required for updating apt package) branch, destroy the existing   QA-net, repeat the [build](#build) and [deploy](#deploy) steps. Perform the [sanity checks](#sanity-check) and verify the fix | | |
```

Comment:
I think we should also include making sure that the hotfixes are merged back into develop.